ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+                                        <desc type="OwnershipNote"/>
+                                          <q xml:lang="gez">ይትባርክ᎓ እግዚአብሔር፡ አምላከ᎓ እስራኤል᎓ ዝመጽሐፍ᎓
+                                            ለአባ᎓ አሕዋን᎓ no. 30</q>
+                                           <note>The note is written in a crude hand, in the upper margin,
+                                             in two column. It mentions an owner,
+                                             <persName ref="PRS12914Ahwan" role="owner">the Falāšā manʾabbā ʾAḥwān</persName> from whom
+                                             <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
+                                             bought the manuscript. The number 30 could have been added later.</note>
+                                         </item>
+
+                                         <item xml:id="a4">
+                                           <locus from="#202va" to="203ra"/>
+                                            <desc type="GuestText"/>
+                                              <q xml:lang="gez"></q>
+                                              <note>The note appears as a continuation of <ref type="mspart" corresp="ms_i8"/>,
+                                                added at a later time, in a less professional hand (the <foreign>tamallas</foreign>-sign
+                                                sing inticates that it goes on on the next page).
+                                                It tells the chronology starting from King Yosṭos
+                                              <date>(r. 1711-16)</date> to King Takla Hāymānot, and stops on the
+                                              evens from ca. <date>1772</date>.
+                                              .</note>
+                                              </item>
+
+                                              <item xml:id="a5">
+                                                <locus target="#202ra"/>
+                                                <desc type="DonationNote"/>
+                                                <q xml:lang="gez">ዝመጽሐፍ᎓ ዝ<persName ref="PRS12916MazraetaKrestos">መዝራዕተ᎓ ክርስቶስ᎓</persName> ወብእሲቱ᎓ ምልእተ᎓ ጸጋ᎓ ወውሉዱ᎓
+                                                  ፍስሐ᎓ ክርስቶስ᎓ ሐዋርያ᎓ ክርስቶስ᎓ ስብስትያኖስ᎓ አምደ᎓ ሚካኤል᎓ ዘአጥረዮ᎓ በንዋዩ᎓ ከመ᎓ ይኩን᎓ ለመድኀኒተ᎓ ነፍሱ᎓ ዘሰረቆ᎓
+                                                  <gap reason="omitted"></gap>.</q>
+                                                <note> The note is written in the main hand.</note>
+                                              </item>
+
+                                              <item xml:id="e1">
+                                                <desc type="StampExlibris"/>
+                                                <note>Two labels of the Biblithèque nationale on the front board, poorly readable, on the inner side
```

Comment:
Bibliothèque (or just use the ID to avoid it...)

Reply:
Ok